src/bundlers/node/nodeJsBinaryDependenciesBundler.ts

```diff
@@ -66,11 +71,13 @@ export class NodeJsBinaryDependenciesBundler implements BundlerInterface {
 
         for (const dependency of binaryDependencies) {
             try {
-                const { stdout, stderr } = await exec(
-                    "npx node-pre-gyp --update-binary --fallback-to-build --target_arch=arm64 --target_platform=linux --target_libc=glibc clean install " +
-                        dependency.name,
-                    { cwd: dependency.path },
-                );
+                const baseCommand =
+                    architecture == "arm"
+                        ? "npx node-pre-gyp --update-binary --fallback-to-build --target_arch=arm64 --target_platform=linux --target_libc=glibc clean install "
+                        : "npx node-pre-gyp --update-binary --fallback-to-build --target_arch=amd64 --target_platform=linux --target_libc=glibc clean install ";
+                const { stdout, stderr } = await exec(baseCommand + dependency.name, {
```

Comment:
Can you please replace this old `exec` with the new `execa` library? We are trying to make a slow transition when possible to `execa` because it's designed to be cross platform. Here is an example:
https://github.com/Genez-io/genezio/blob/main/src/packageManagers/npm.ts#L9

Reply:
Done, looks better, thanks!